docs/operating/troubleshooting.md

```diff
@@ -5,4 +5,35 @@ menu: operating
 slug: /troubleshooting.md
 ---
 
-Work in progress (:
+# Troubleshooting; Common cases
+
+
+## Overlaps
+
+Overlaps should never happen in healthy and well configured Thanos system. That's why there is no automatic repair.
```

Comment:
```suggestion
**Block overlap**: Set of blocks with exactly the same external labels in meta.json and for the same time or overlapping time period.

Thanos is designed to never end up with overlapped blocks. This means that (uncontrolled) block overlap should never happen in a healthy and well configured Thanos system. That's why there is no automatic repair for this. Since it's an unexpected incident:
* All reader components like Store Gateway will handle this gracefully (overlapped samples will be deduplicated).
* Thanos compactor will stop all activities and HALT or crash (with metric and will error log). This is because it cannot perform compactions and downsampling. In the overlap situation, we know something unexpected happened (e.g manual block upload, some malformed data etc), so it's safer to stop or crash loop (it's configurable).
```